pkg/controllers/provisioning/v1alpha1/allocator/filter.go

```diff
@@ -17,8 +17,7 @@ package allocator
 import (
 	"context"
 	"fmt"
-	"github.com/awslabs/karpenter/pkg/controllers/provisioning/v1alpha1/util/scheduling"
-
+	podUtil "github.com/awslabs/karpenter/pkg/utils/pod"
```

Comment:
This package naming doesn't follow the style guide: https://blog.golang.org/package-names. utils/scheduling is probably the right name.

Reply:
Thanks for the resource!